lib/rubycritic/core/analysed_module.rb

```diff
@@ -19,6 +23,22 @@ def path
       @path ||= pathname.to_s
     end
 
+    def file_location
+      file_path[0...-1].join('/')
+    end
+
+    def file_name
+      file_path[-1]
+    end
+
+    def file_path
+      path.split('/')
+    end
+
+    def line_count
+      `wc -l "#{path}"`.strip.split(' ')[0].to_i
```

Comment:
This is introducing extra behaviour that may not work on other platforms, I believe.

Maybe this could be moved to another PR that attempts to solve this in a more cross-platform way?